cachix/src/Cachix/Client/Push.hs

```diff
@@ -77,95 +76,81 @@ pushSingleStorePath
   -> Text -- ^ store path
   -> m r -- ^ r is determined by the 'PushStrategy'
 pushSingleStorePath ce _store cache cb storePath = retryPath $ \retrystatus -> do
-
   let (storeHash, storeSuffix) = splitStorePath $ toS storePath
       name = pushCacheName cache
 
```

Comment:
If you remove the presence check from `pushSingleStorePath`, doesn't that make cachix overwrite store paths, for example when using `--watch-store`?
We may have to split this function into one that checks (old name) and one that doesn't.

Reply:
Ugh, need to fix that.